crates/oxc_linter/src/snapshots/no_jsx_as_prop.snap

```diff
@@ -0,0 +1,26 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: no_jsx_as_prop
+---
+  ⚠ eslint-plugin-react-perf(no-jsx-as-prop):
+   ╭─[no_jsx_as_prop.tsx:1:1]
+ 1 │ <Item jsx={<SubItem />} />
+   · ──────────────────────────
```

Comment:
Can you pin point the span to the actual error? 

https://oxc-project.github.io/docs/contribute/linter.html#general-advice

> We want the user to focus on the problematic code rather than deciphering the error message to identify which part of the code is erroneous.

Reply:
👌